.github/workflows/build_and_test.yml

```diff
@@ -383,6 +383,10 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: ${{ matrix.java }}
+    - name: Install Python packages (Python 3.9, PyPy3)
+      run: |
+        # To test excel I/O for pandas API on Spark.
+        python3.9 -m pip install openpyxl
```

Comment:
can we add this into Dockerfile? https://github.com/apache/spark/blob/master/dev/infra/Dockerfile

Reply:
Sounds good! Just removed